js/layout-javascript/news-feed-code.js

```diff
@@ -284,6 +285,41 @@ DynamicList.prototype.attachObservers = function() {
         }, 0);
       }
     })
+    .on('click change keydown', '.hidden-filter-controls-filter-date-range', function(event) {
+      event.stopPropagation();
+
+      if (!_this.Utils.accessibilityHelpers.isExecute(event)) {
+        return;
+      }
+
+      var $filter = $(this);
+      var $resetDates = $('#resetDate');
+      var value = $filter.val().trim();
+      var formatDate = moment(value).format(_this.DATE_FORMAT);
+
+      $resetDates.removeClass('hidden');
+      $filter.attr('data-value', formatDate);
+
+      Fliplet.Analytics.trackEvent({
+        category: 'list_dynamic_' + _this.data.layout,
+        action: 'filter',
+        label: $filter.text().trim()
+      });
+
+      _this.toggleFilterElement($filter);
+
+      if ($filter.parents('.inline-filter-holder').length) {
+        // @HACK Skip an execution loop to allow custom handlers to update the filters
+        setTimeout(function() {
+          _this.searchData();
+        }, 0);
+      }
+    })
+    .on('click', '#resetDate', function() {
+      $('input.date-picker').datepicker('setDate', new Date());
```

Comment:
[Requirement](https://weboo.atlassian.net/browse/OD-550) says:

```
Scenario 2.6: Resetting filter by date range values

GIVEN the user selects the date "FROM"  and the date "TO" values

WHEN the user clicks the "Reset" link 

THEN the user sees the "FROM" and "TO" date pickers default to the minimum and maximum values found in the data loaded into the LFD 

AND the "Reset" link hides itself
```

This looks like the reset button is resetting the date pickers to the current date instead of to the minimum and maximum values found in the data loaded into the LFD.